hack/.golint_failures

```diff
@@ -211,7 +211,6 @@ pkg/kubelet/dockershim/network/hostport/testing
 pkg/kubelet/dockershim/network/kubenet
 pkg/kubelet/dockershim/network/testing
 pkg/kubelet/events
-pkg/kubelet/images
```

Comment:
@Pingan2017 thanks for contributing. what's the purpose of this pull plz? these changes are impacting readability of blame msg

Reply:
@dims it'll be tons of pulls like this flooding the repo. how about put all the changes into one single large pull?

Reply:
@yue9944882 one large single pull is not desirable. i'd prefer contributors picking off a few packages at a time and filing a PR

Reply:
>  i'd prefer contributors picking off a few packages at a time and filing a PR

@dims +1. take `pkg/kubelet` as example, it has ~60 more pkgs to go. i guess we might possibly make it within 5~10 pulls? wdyt?

Reply:
@yue9944882 +1, feel free to suggest groups of packages for the new folks in the original issue.